### ISSUE TITLE
Error loading qml file

### ISSUE THREAD

**Issue by killerjn** [thevladsoft/BeClock-plasmoid#1]
![323](https://user-images.githubusercontent.com/32747059/38700386-7a024132-3e60-11e8-8586-f7f40fae981f.png)

**Comment by thevladsoft** [comment]
Yes, it's based on the Digital Clock of plasma 5.8, so I'm pretty sure you need at least that version (I use it on 5.12 without problem.)

**Comment by killerjn** [comment]
update to 5.8 and it still does not work

**Comment by thevladsoft** [comment]
It's weird. Can you use the original Digital Clock plasmoid? As this is based on that plasmoid it depends on the same libraries.

Also, if you have plasmoidviewer (on debian and derivates should be in the plasma-sdk package) please run in a terminal:

    plasmoidviewer -a BeClock-plasmoid

and copy the exit of the command here.

**Comment by killerjn** [comment]
![screenshot_20180419_145001](https://user-images.githubusercontent.com/32747059/39014828-02e70548-43e1-11e8-890d-0a211fecb5dd.png)

**Comment by thevladsoft** [comment]
Is that all the text it returns? Could you please copy the text and not an screenshot.

I can't reproduce your problem, so I don't have enough info to help you right now.

**Comment by killerjn** [comment]
![peek 20-04-2018 21-29](https://user-images.githubusercontent.com/32747059/39079534-48f387a8-44e2-11e8-80aa-0b6ee1e8d08d.gif)

alx@ubs-MS-7592:~$ plasmoidviewer -a BeClock-plasmoid
QML debugging is enabled. Only use this in a safe environment.
qml: View QML loaded
Toolbox not loading, toolbox package is either invalid or disabled.
qml: New Containment: ContainmentInterface(0x15ec8b0)
Constructing a KPluginInfo object from old style JSON. Please use kcoreaddons_desktop_to_json() for "/home/alx" instead of kservice_desktop_to_json() in your CMake code.
alx@ubs-MS-7592:~$
